lib/LaTeXML/Core/Token.pm

```diff
@@ -251,15 +251,15 @@ sub neutralize {
 
 # Mark a token as not to be expanded (\noexpand) by hiding itself as the 3rd element of a new token.
 # Wonder if this should only have effect on expandable tokens?
-sub noexpand {
+sub with_dont_expand {
   my ($self) = @_;
   return bless [$$self[0], $$self[1], $self], 'LaTeXML::Core::Token'; }
 
 # Return the original token of a not-expanded token,
 # or undef if it isn't marked as such.
-sub is_notexpanded {
+sub is_dont_expand {
   my ($self) = @_;
-  return $$self[2]; }
+  return $$self[2] ? 1 : 0; }
 
```

Comment:
I guess I like your renaming generally; more consistent. But I still think I'd rather have an accessor to the token, so how about renaming "is_dont_expand" to "get_dont_expand" and just returning ```$$self[2]```

Reply:
sounds good